subprojects/docs/src/docs/userguide/groovy_plugin.adoc

```diff
@@ -224,19 +224,26 @@ The Groovy plugin adds a link:{groovyDslPath}/org.gradle.api.tasks.compile.Groov
 
 _Caveat: Groovy compilation avoidance is an incubating feature since Gradle 5.6. There are known inaccuracies so please enable it at your own risk._
 
-To enable the incubating support for Groovy compilation avoidance, add a system property `org.gradle.groovy.compilation.avoidance` to your Gradle invocation, for example:
+To enable the incubating support for Groovy compilation avoidance, add a line to your settings file:
```

Comment:
I was hoping we have some documentation about the feature preview API. Though it does not seem to be the case...

Can you maybe link to the API doc of `enableFeaturePreview` at least?

Reply:
The documentation has been per feature. So it is good as it is now.